src/patternfly/components/Popover/examples/Popover.md

```diff
@@ -132,34 +140,126 @@ import './Popover.css'
+| `aria-modal="true"`                                   | `.pf-c-popover`                                                       | Tells assistive technologies that the windows underneath the current popover are not available for interaction. **Required**                                                                                                                                             |
+| `aria-label="Close"`                                  | `.pf-c-button`                                                        | Provides an accessible name for the close button as it uses an icon instead of text. **Required**                                                                                                                                                                        |
+| `aria-hidden="true"`                                  | Parent element containing the page contents when the popover is open. | Hides main contents of the page from screen readers. The element with `.pf-c-popover` must not be a descendent of the element with `aria-hidden="true"`. For more info see [trapping focus](https://pf4.patternfly.org/accessibility-guide#trapping-focus). **Required** |
 
 ### Usage
-| Class | Applied to | Outcome |
-| -- | -- | -- |
-| `.pf-c-popover` | `<div>` |  Creates a popover. Always use it with a modifier class that positions the popover relative to the element that triggered it. **Required** |
-| `.pf-c-popover__arrow` | `<div>` |  Creates an arrow pointing towards the element the popover describes. **Required** |
-| `.pf-c-popover__content` | `<div>` |  Creates the content area of the popover. **Required** |
-| `.pf-c-button` | `<button>` |  Positions the close icon in the top-right corner of the popover. **Required** |
-| `.pf-c-title` | `<h1>`,`<h2>`,`<h3>`,`<h4>`,`<h5>`,`<h6>` |  Initiates a title. Always use it with a modifier class. See [title component](/documentation/core/components/title) for more info.|
-| `.pf-c-popover__body` | `<div>` |  The popover's body text. **Required** |
-| `.pf-c-popover__footer` | `<footer>` | Initiates a popover footer. |
-| `.pf-m-left` | `.pf-c-popover` | Positions the popover to the left of the element. |
-| `.pf-m-right` | `.pf-c-popover` | Positions the popover to the right of the element. |
-| `.pf-m-top` | `.pf-c-popover` | Positions the popover to the top of the element. |
-| `.pf-m-bottom` | `.pf-c-popover` | Positions the popover to the bottom of the element. |
-| `.pf-m-no-padding` | `.pf-c-popover` | Removes the outer padding from the popover content. |
-| `.pf-m-width-auto` | `.pf-c-popover` | Allows popover width to be defined by the popover content. |
+
+| Class                    | Applied to                                | Outcome                                                                                                                                   |
```

Comment:
We used to format our markdown tables this way but found the maintenance to maintain the whitespace more work than it's worth, and just use single spaces in the tables now. You can see that across the library in our other examples. Would you mind being consistent with the markdown tables in the rest of the component documentation

Reply:
This is fixed now. I had a setting in VSCode which was formatting the code.